### PR TITLE
Adds `mod` to Aside

### PR DESCRIPTION
**Pull request by peter1000**
https://github.com/MichaelHatherly/Lexicon.jl/pull/116#issuecomment-108828801

### CODE REVIEW

**Comment by MichaelHatherly** [comment]
"ExampleAside.jl" file is missing.

**Comment by peter1000** [comment]
sorry added missing file

File: src/Collector/docstrings.jl [hunk 1]
```diff
@@ -74,10 +74,9 @@ Extract the comment block from expressions and capture metadata.
 """
 function get_aside!(output, moddata, state, file, block)
     line, comment, _, _ = block
```
Comment (MichaelHatherly) [inline src/Collector/docstrings.jl]: Deleted line break.

**Comment by peter1000** [comment]
Do you want me to skip tests for julia 0.3 ?

**Comment by MichaelHatherly** [comment]
b4146e22b06e51fff6b77290296ea1ab09f60f0a should make your test work on 0.3.

**Comment by peter1000** [comment]
thanks will check that.

**Pull request merged**